src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -79,7 +80,7 @@ export interface Order2CheckoutModel {
   isLoading: boolean
   /** Order is redirecting to the details page */
   expressCheckoutSubmitting: boolean
-  loadingError: CheckoutLoadingError | null
+  criticalCheckoutError: CriticalCheckoutError | null
```

Comment:
i wonder if there is a better name for this - or if all modals should use the same component and we just match on the errors/content passed in.